src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -199,6 +200,7 @@ export class Kernel implements IKernel {
         const sessionPromise = this.startNotebook().then((nb) => nb.session);
         const promise = this.kernelExecution.executeCell(sessionPromise, cell);
         this.trackNotebookCellPerceivedColdTime(stopWatch, sessionPromise, promise).catch(noop);
+        void promise.then((state) => traceInfo(`Cell ${cell.index} executed with state ${state}`));
```

Comment:
Logging so we know cells completed (today we log when we start cells but not whether it completed or not)
Was crucial for a user issue.